src/components/Form/ProjectsDataDatePicker.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import DateView from 'react-datepicker';
```

Comment:
DatePicker is a better name than DateView

Reply:
Resolved